app.py

```diff
@@ -204,6 +222,14 @@ def discover():
     )
 
 
+@app.route('/promoted/')
```

Comment:
Is there meant to be this separate page for those, or should 'promoted' snaps be displayed on some other existing page (search, discover?)? @Greggless?

Reply:
On phase 2 of /discover (https://github.com/CanonicalLtd/snapcraft-design/issues/197) we link to a page using `see more snaps`. This is the core of that page, a listing of all 'promoted snaps. 

It needs some design love, mainly including a search in the hero. @tbille put this together quickly last night (thanks!)

Reply:
This should be changed to `/snaps/`.